tensorflow/python/keras/integration_test/gradient_checkpoint_test.py

```diff
@@ -0,0 +1,160 @@
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import tensorflow as tf
+from tensorflow.keras import layers, optimizers
```

Comment:
Can you use:

layers = tf.keras.layers
optimizers = tf.keras.optimizers

it's a tensorflow.python.keras vs. tensorflow.keras issue otherwise. If we always go through the public API I think it'll be fine.

Reply:
Done. Thanks!